.github/workflows/check-links.yml

```diff
@@ -51,8 +50,8 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: Create Issue From File
-      uses: peter-evans/create-issue-from-file@v3
-      with:
-        title: Link Checker Report
-        content-filepath: ./lychee/out.md
+    #- name: Create Issue From File
+    #  uses: peter-evans/create-issue-from-file@v3
+    #  with:
+    #    title: Link Checker Report
```

Comment:
Include the date in the PR title like in https://github.com/GenericMappingTools/gmt/pull/5576?